### ISSUE TITLE
openuserjs down for me

### ISSUE THREAD

**Issue by rezad1393** [ChaseChoi/doi2scihub#15]
I get the error that it is down for me and so violentmonkey gives error when checking for updates and see that it cant get this:
https://openuserjs.org/src/libs/sizzle/GM_config.js

https://www.isitdownrightnow.com/openuserjs.org.html

btw can you put this file on ipfs so that it stays there forever? I live in Iran and internet can be spotty here, so using something like ipfs helps a lot for getting files like this.

**Comment by ChaseChoi** [comment]
It seems that the result of [https://www.isitdownrightnow.com/openuserjs.org.html](https://www.isitdownrightnow.com/openuserjs.org.html) is **not** reliable. The file on [https://openuserjs.org/src/libs/sizzle/GM_config.js](https://openuserjs.org/src/libs/sizzle/GM_config.js) is accessible in my country, but it's still telling me that it's down for everyone.
Yes. It would be a good idea to put the file on somewhere more reliable.

**Comment by rezad1393** [comment]
the issue is that it is not even accesebale via Tor proxy, as in it doesnt even tcp response.
if you are familiar with ipfs maybe if you can upload it to ipfs on cloudflare and then change the address to that?

**Comment by ChaseChoi** [comment]
OK. I will try that.

**Comment by rezad1393** [comment]
> OK. I will try that.

thanks.
I prefers ipfs because it is not domain dependant.
and I said cloudflare so that maybe by pinging the resource it stays up in ipfs. but I think that is readonly for web users and write for their customers.


you can use this maybe too.
https://raw.githubusercontent.com/sizzlemctwizzle/GM_config/master/gm_config.js
or import it in your own github here and use that.

**Comment by ChaseChoi** [comment]
I've released [v1.0.30](https://greasyfork.org/en/scripts/412498-doi-to-sci-hub), give it a try

**Comment by rezad1393** [comment]
what is this?
cdn.jsdelivr.net

**Comment by ChaseChoi** [comment]
It's a free CDN for open source projects. Can you access the file as expected?
https://raw.githubusercontent.com/sizzlemctwizzle/GM_config/master/gm_config.js is quite slow for users in China, so I use jsdelivr

**Comment by rezad1393** [comment]
yes.
jsdelivr ??
how did you upload to it?

**Comment by rezad1393** [comment]
> It's a free CDN for open source projects. Can you access the file as expected? https://raw.githubusercontent.com/sizzlemctwizzle/GM_config/master/gm_config.js is quite slow for users in China, so I use jsdelivr

is github filtered in china?
or just slow.?
I recommended a host that can support ipfs because that is hash based file indentifier instead of location based.
are you familiar with ipfs?
I think things like that are needed for static files like css and libraries .

**Comment by ChaseChoi** [comment]
`https://raw.githubusercontent.com/*` is blocked in China. 
I'm not familiar with **ipfs**, so I find the easiest way to make sure the resource is accessible to you as soon as possible. Maybe you could create a merge request.

**Comment by rezad1393** [comment]
ipfs is cool, I recommend reading about it a bit.
basically it is kind like bittorrentv2, for web.
web3 without the scams.
files are hash based distributed.
https://teddit.net/r/ipfs/comments/7mkqi1/eli5_ipfs/
